swatch-core/src/main/resources/logback-spring.xml

```diff
@@ -39,4 +39,34 @@
         <appender-ref ref="ConsoleAppender" />
         <appender-ref ref="SplunkAppender" />
     </root>
+
+    <if condition='property("ENABLE_SPLUNK_HEC").equals("true")'>
+        <then>
+            <appender name="http" class="com.splunk.logging.HttpEventCollectorLogbackAppender">
+                <url>${SPLUNK_HEC_URL}</url>
```

Comment:
Are these getting pulled from the environment?  Looking at the [docs](https://logback.qos.ch/manual/configuration.html#variableSubstitution) it looks like it pulls from Java system properties instead?  Same question on the `<if condition='property` bit.

One thought is that we could use the `<property file=` syntax and load the properties from a config map that we write.  That would actually cut down on a lot of the redundancy in the clowder config since instead of redefining the environment variables over and over again, we could just mount in the same properties file into each one.

Reply:
"During substitution, properties are looked up in the local scope first, in the context scope second, in the system properties scope third, and in the OS environment fourth and last."  http://logback.qos.ch/manual/configuration.html#scopes